Cargo.toml

```diff
@@ -35,3 +34,7 @@ image = { version = "0.25.0", default-features = false, features = [
 
 [target.'cfg(target_arch = "wasm32")'.dev-dependencies]
 wasm-bindgen-test = "0.3.42"
+
+[features]
+# Enables the wgpu feature on vello, which is disabled by default
+vello-wgpu = ["vello/wgpu"]
```

Comment:
What's the reason for this `vello-` prefix?

Reply:
The feature does not alter the behaviour or features provided by this crate in any way, only on the crate it reexports.

Reply:
I think @DJMcNab 's callout is valid - this is an anti-pattern.
Let's change it to just `wgpu` and then we're good to merge.